tools/arch-visualizer/viewer/src/utils/search.ts

```
@@ -0,0 +1,86 @@
+import Fuse from "fuse.js";
+import type { Architecture, Component, FileInfo, Symbol } from "../types";
```

Biome (2.3.13)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)